WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -758,8 +759,10 @@ nick_of_a_thousand_faces = {
 		age = 16
 		intrigue = 10
 		trait = disfigured
-		trait = deceitful
-		trait = envious
+		OR = {
+			trait = deceitful
+			trait = envious
+		}
```

Comment:
It is not a mistake. Maybe you dislike how it is, but isn't an error, it is intended to have an emphasis on both traits.